test/status.bats

```diff
@@ -3,102 +3,95 @@
 load helpers
 
 function setup() {
-    setup_test
-    start_crio
+	setup_test
+	start_crio
 }
 
 function teardown() {
-    cleanup_test
+	cleanup_test
 }
 
 function run_crio_status() {
-    run "${CRIO_STATUS_BINARY_PATH}" $@
+	run "${CRIO_STATUS_BINARY_PATH}" "$@"
+	echo "$output"
 }
 
 @test "status should fail if no subcommand is provided" {
-    # when
-    run_crio_status
-    echo "$output"
+	# when
+	run_crio_status
 
-    # then
-    [ "$status" -eq 1 ]
+	# then
+	[ "$status" -eq 1 ]
 }
 
 @test "status should succeed to retrieve the config" {
-    # when
-    run_crio_status --socket="${CRIO_SOCKET}" config
-    echo "$output"
+	# when
+	run_crio_status --socket="${CRIO_SOCKET}" config
 
-    # then
-    [ "$status" -eq 0 ]
-    [[ "$output" == *"[crio]"* ]]
+	# then
+	[ "$status" -eq 0 ]
+	[[ "$output" == *"[crio]"* ]]
 }
 
 @test "status should fail to retrieve the config with invalid socket" {
-    # when
-    run_crio_status --socket wrong.sock c
-    echo "$output"
+	# when
+	run_crio_status --socket wrong.sock c
 
-    # then
-    [ "$status" -eq 1 ]
+	# then
+	[ "$status" -eq 1 ]
 }
 
 @test "status should succeed to retrieve the info" {
-    # when
-    run_crio_status --socket="${CRIO_SOCKET}" info
-    echo "$output"
+	# when
+	run_crio_status --socket="${CRIO_SOCKET}" info
 
-    # then
-    [ "$status" -eq 0 ]
-    [[ "$output" == *"storage driver"* ]]
+	# then
+	[ "$status" -eq 0 ]
+	[[ "$output" == *"storage driver"* ]]
 }
 
 @test "status should fail to retrieve the info with invalid socket" {
-    # when
-    run_crio_status --socket wrong.sock i
-    echo "$output"
+	# when
+	run_crio_status --socket wrong.sock i
 
-    # then
-    [ "$status" -eq 1 ]
+	# then
+	[ "$status" -eq 1 ]
 }
 
 @test "succeed to retrieve the container info" {
-    # given
-    run crictl runp "$TESTDATA"/sandbox_config.json
-    echo "$output"
-    [ "$status" -eq 0 ]
-    pod="$output"
-    run crictl create "$pod" "$TESTDATA"/container_redis.json "$TESTDATA"/sandbox_config.json
-    echo "$output"
-    [ "$status" -eq 0 ]
-    ctr="$output"
-    run crictl start "$ctr"
-    echo "$output"
-    [ "$status" -eq 0 ]
-
-    # when
-    run_crio_status --socket="${CRIO_SOCKET}" containers --id "$ctr"
-    echo "$output"
-
-    # then
-    [ "$status" -eq 0 ]
-    [[ "$output" == *"sandbox: $pod"* ]]
+	# given
+	run crictl runp "$TESTDATA"/sandbox_config.json
```

Comment:
Don't we want to remove the `run` here and in the following few lines as well?

Reply:
Yes. I lost that commit during the rebase :-\

Let me redo this

Reply:
done; thanks for noticing this!